examples/provisioner/provisioner-100-cpu.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: karpenter.sh/v1alpha5
+kind: Provisioner
+metadata:
+  name: default
+spec:
+  architectures:
```

Comment:
Did you test this spec? This looks like v1alpha3 :P

Reply:
I tested all the workloads and _most_ of the provisioners. :grimacing: Let me double check on these.